src/project.test.js

```diff
@@ -2,23 +2,34 @@ const { project } = require('../');
 
 test('valid projection', () => {
   expect(
-    project({
-      forestEcoregion: '1',
-      additional: 'unknown',
-      heightLevel: 'SA',
-      forestType: '60*',
-      slope: '<70',
-      tannenareal: 'unknown',
-      relief: 'unknown',
-    }).target,
-  ).toBe('50*');
+    project(
+      {
+        forestEcoregion: '1',
+        additional: 'unknown',
+        heightLevel: 'OSA',
+        forestType: '59L',
+        slope: 'unknown',
+        tannenareal: 'unknown',
+        relief: 'unknown',
+      },
+      'UM',
+    ).forestType,
+  ).toBe('1');
 });
 
 test('invalid location values', () => {
   expect(() =>
     project({ forestType: '60*', forestEcoregion: 'fooBar' }),
   ).toThrowError('fooBar for forestEcoregion is not valid.');
 
+  // expect(() =>
```

Comment:
Please only add running tests.

Reply:
removed the commented one. All other tests are running.